packages/calcite-components/src/components/tabs/tabs.scss

```diff
@@ -1,26 +1,72 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-tab-accent-color-hover: Specifies the component's accent color when hovered.
+ * @prop --calcite-tab-accent-color-selected: Specifies the component's accent color when selected.
+ * @prop --calcite-tab-background-color-hover: Specifies the component's background color when hovered.
+ * @prop --calcite-tab-background-color: Specifies the component's background color.
+ * @prop --calcite-tab-content-space-y: Specifies the block padding of the component's content.
+ * @prop --calcite-tab-icon-color: Specifies the component's icon color.
```

Comment:
Do we need this if we offer the explicit `start` and `end` ones?

Reply:
Good question. I'm open to simplifying, but having the generic icon color prop aligns with others (e.g., `accordion-item`, `button`) and is a convenience. cc @alisonailea